tests/test_api_key_tools.py

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
If we add the following line to the top of this file

```bash
#!/usr/bin/env python
```

And then call `chmod +x tools/api_key_tool.py`

the user will be able to use this file as a standalone script by typing

```bash
./tools/api_key_tool.py
```

However, if I haven't explained this well, no worries, I can take care of it in a followup PR.